src/Zonotope.jl

```diff
@@ -684,3 +685,36 @@ function translate(Z::Zonotope{N}, v::AbstractVector{N}; share::Bool=false
     generators = share ? Z.generators : copy(Z.generators)
     return Zonotope(c, generators)
 end
+
+"""
+    overapproximate(Z::Zonotope)
+
+Reduce the order of a given Zonotope to 1
+
+### Input
+
+- `Z`     -- zonotope
+
+### Output
+
+A overapproximation of the given zonotope using a Parallelotope.
+
+### Algorithm
+
+The algorithm is based on propositions discussed in Section 5 of [this paper](https://mediatum.ub.tum.de/doc/1287515/893367.pdf)
+"""
+function overapproximate(Z::Zonotope)
+    p, n = ngens(Z), dim(Z)
+    if p == n
+        return Z
+    elseif p > n
+        V = 1:n # could use difference selection criteria
+    else
+        error("the zonotope order is $(order(Z)) but it should be at least 1")
```

Comment:
I think it is better to do:
```julia
if p == n
    return Z
elseif p < n
    error(...)
```
You actually do not need `V` at all.

Reply:
Here `V`  refers to the generators that are selected. The simplest choice is to choose the first `n` of them, that is why this variable `V` is needed. At a second stage, we will implement heuristics for choosing the generators according to some criterion.